src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -1342,8 +1320,8 @@ void GLHEVert::calcShortTimestepGFunctions(EnergyPlusData &state)
         // calculate bh wall temp
         Real64 T_bhWall = 0.0;
         for (int cell_index = 0; cell_index < num_cells; ++cell_index) {
-            auto &leftCell = Cells[cell_index];
-            auto &rightCell = Cells[cell_index + 1];
+            auto const &leftCell = Cells[cell_index];
+            auto const &rightCell = Cells[cell_index + 1];
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:1345]:(style),[constVariable],Variable 'leftCell' can be declared as reference to const
[src/EnergyPlus/GroundHeatExchangers.cc:1346]:(style),[constVariable],Variable 'rightCell' can be declared as reference to const